ksql-engine/src/test/resources/expected_topology/5_3_0_pre/joins_-_stream_stream_inner_join

```diff
@@ -0,0 +1,80 @@
+{
+  "ksql.extension.dir" : "ext",
+  "ksql.streams.cache.max.bytes.buffering" : "0",
+  "ksql.transient.prefix" : "transient_",
+  "ksql.named.internal.topics" : "on",
+  "ksql.windowed.session.key.legacy" : "false",
+  "ksql.schema.registry.url" : "http://localhost:8081",
+  "ksql.streams.default.deserialization.exception.handler" : "io.confluent.ksql.errors.LogMetricAndContinueExceptionHandler",
+  "ksql.output.topic.name.prefix" : "",
+  "ksql.streams.auto.offset.reset" : "earliest",
+  "ksql.sink.partitions" : null,
+  "ksql.avro.maps.named" : "true",
+  "ksql.service.id" : "some.ksql.service.id",
+  "ksql.streams.state.dir" : "/var/folders/2d/3pt97ylj3zngd51bwl91bl3r0000gp/T/kafka-6208056045361781873",
+  "ksql.internal.topic.replicas" : "1",
+  "ksql.streams.default.production.exception.handler" : "io.confluent.ksql.errors.ProductionExceptionHandlerUtil$LogAndFailProductionExceptionHandler",
+  "ksql.streams.bootstrap.servers" : "localhost:0",
+  "ksql.streams.commit.interval.ms" : "2000",
+  "ksql.streams.auto.commit.interval.ms" : "0",
+  "ksql.sink.replicas" : null,
+  "ksql.streams.topology.optimization" : "all",
+  "ksql.streams.num.stream.threads" : "4",
+  "ksql.udfs.enabled" : "true",
+  "ksql.udf.enable.security.manager" : "true",
+  "ksql.functions.substring.legacy.args" : "false",
+  "ksql.streams.application.id" : "some.ksql.service.id",
+  "ksql.sink.window.change.log.additional.retention" : "1000000",
+  "ksql.udf.collect.metrics" : "false",
+  "ksql.persistent.prefix" : "query_",
+  "ksql.query.persistent.active.limit" : "2147483647"
+}
+CONFIGS_END
+CSAS_INNER_JOIN_0.KafkaTopic_Left.source = struct<ID optional<int64>,NAME optional<string>,VALUE optional<int64>>
+CSAS_INNER_JOIN_0.KafkaTopic_Right.source = struct<ID optional<int64>,F1 optional<string>,F2 optional<int64>>
+CSAS_INNER_JOIN_0.Join.left = struct<ROWTIME optional<int64>,ROWKEY optional<string>,ID optional<int64>,NAME optional<string>,VALUE optional<int64>>
```

Comment:
for my understanding, why do we explicitly append `ROWTIME optional<int64>,ROWKEY optional<string>` to the Join contexts but not to the others?

Reply:
That would be a very good question!  My guess would be it was not intentional. Unfortunately, removing it is a none-backwards compatible change.  This is the kind of quirk that make me add these tests in the first place.